src/packages/README.md

```diff
@@ -9,17 +9,81 @@ They have different levels of completeness, maturity, security, and support.
 release, which differs substantially from prior releases and software
 snapshots due to the introduction of a `pkg64c` package set.**
 
+## Overview
+
+The following table presents an overview of available package managers in
+CheriBSD that are described in more details in consecutive sections.
+
+| ABI        | #      | Manager | Install path   | Suitable for     | Examples |
+|------------|--------|---------|----------------|------------------|----------|
+| Hybrid ABI | 20,000 | `pkg64` | `/usr/local64` | Day-to-day use   | bash<br/>gdb-cheri<br/>llvm-base<br/>vim |
+| CheriABI   | 8,000  | `pkg64c`| `/usr/local`   | Experimental use | dash<br/>git<br/>nano<br/>rsync |
+
+## Package managers in CheriBSD
+
+*Note:* As of this writing we only provide packages for Morello systems.
+We aim to add CHERI-RISC-V package sets in the near future.
+
+CheriBSD includes two package managers:
+* `pkg64` for hybrid ABI packages;
+* `pkg64c` for CheriABI packages.
+
+The FreeBSD package manager `pkg` is not available on CheriBSD.
```

Comment:
Technically it's the base system bootstrapping wrapper that isn't available. It's installed as (e.g. `/usr/local64/sbin/pkg`)

Reply:
By the FreeBSD package manager, I mean `pkg` from FreeBSD and FreeBSD packages from pkg.FreeBSD.org. A user cannot use any FreeBSD packages as they would be installed in `/usr/local` and conflict with CheriABI packages.

`/usr/local64/sbin/pkg` is almost the same as the FreeBSD's pkg but shouldn't be advertised as it because it doesn't actually use packages from pkg.FreeBSD.org.